src/Analyzers/CSharp/Tests/NewLines/EmbeddedStatementPlacement/EmbeddedStatementPlacementTests.cs

```diff
@@ -298,6 +299,43 @@ void M()
         if (true)
             return;
     }
+}";
+            await new VerifyCS.Test
+            {
+                TestCode = source,
+                FixedCode = fixedCode,
+                Options = { { CSharpCodeStyleOptions.AllowEmbeddedStatementsOnSameLine, CodeStyleOptions2.FalseWithSuggestionEnforcement } }
```

Comment:
:bulb: I think this also works:

```suggestion
                Options = { { CSharpCodeStyleOptions.AllowEmbeddedStatementsOnSameLine, false } }
```

Reply:
done.